plugins/block-extension-tooltip/package.json

```diff
@@ -42,7 +42,7 @@
   "devDependencies": {
     "@blockly/dev-scripts": "^1.2.12",
     "@blockly/dev-tools": "2.6.1",
-    "blockly": "5.20210325.1",
+    "blockly": "^7.20211209.0",
```

Comment:
I don't think this can be updated yet because of [this issue](https://github.com/google/blockly-samples/issues/899).